app/controllers/shopify_app/app_proxy_controller.rb

```diff
@@ -0,0 +1,9 @@
+module ShopifyApp
+  class AppProxyController < ApplicationController
+    include ShopifyApp::AppProxyVerification
+
+    skip_before_action :verify_authenticity_token
+
+    before_action :verify_proxy_request
+  end
+end
```

Comment:
hmm I don't really like that this controller doesn't do anything - the webhook controller provided delegates out to jobs and we were able to establish a pattern for doing so automatically which meant that uses of the gem didn't need to modify the controller at all meaning it could live inside the engine with no issues.

This controller will need to be edited so it doesn't make sense as an engine controller. We have 2 options here:
1. Add this controller to a new generator for app proxies and this controller lives in the host app and includes this module (the module should inject the before filter though)
2. I am not really familiar with App Proxies but perhaps we can do some kind of delegation to jobs like the webhook controller.

Let me know if that makes sense.